src/database/schema/ChatSchema.ts

```diff
@@ -249,7 +249,7 @@ class ChatSchema implements BaseDbSchema {
   }
 
   static fromGetAllAnonymousChannelAPI(channelId, json): ChatSchema {
-    let rawJson = null;
+    let rawJson: string | null = null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, this change improves type safety by explicitly stating that `rawJson` can be either a string or null.
```diff
-    let rawJson = null;
+    let rawJson: string | null = null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->